ui/jquery.ui.mouse.js

```diff
@@ -23,10 +23,117 @@
 //JASPERSOFT #1 #3
 define(function(require) {
 
-	var $ = require("./jquery.ui.widget");
-	var isIE = function () {
-		return navigator.appName === "Microsoft Internet Explorer" || navigator.userAgent.indexOf("Trident/") >= 0;
-	};
+    var $ = require("./jquery.ui.widget");
+
+    var isIE = function () {
+        return navigator.appName === "Microsoft Internet Explorer" || navigator.userAgent.indexOf("Trident/") >= 0;
```

Comment:
This may be OK, I think all previous versions of IE return this value "Microsoft Internet Explorer" until IE11 which returns "Netscape" but it may be better to check for a match for "MSIE" in the user-agent instead. Unsure. Again, this can be ignored as it does still fix the underlying issue, but may be a good change to consider in the future.

Reply:
yes, I changed to Netscape, but I tried to keep the logic to stay more wider rather being focused on IE-11 only because maybe we'll all more code there for IE-10